src/pages/index.astro

```diff
@@ -24,7 +26,7 @@ import SecondaryContactSection from "@/components/sections/secondary-contact-sec
     <NewTechnologiesSection />
     <SolutionsSection />
     <ContactSection />
-    <TestimonialsSection />
+    <NewTestimonialsSection />
```

Comment:
We should keep naming it just `TestimonialsSection`. And it's not very important but, we should remove the commented imports too.